### DISCUSSION TITLE
ES900RX Receiver Configuring for 433MHZ

### DISCUSSION THREAD

**Discussion by krupis** [ExpressLRS/ExpressLRS#2879] [General]
I have a ES900RX Receiver (Happymodel ExpressLRS ES900RX Receiver) that contains semtech SX1276 lora chip. It is configured for 868Mhz but from what I understand, SX1276 is also capable of using 433Mhz frequency:

![image](https://github.com/user-attachments/assets/b000fb1d-e767-4193-b130-f279c94e7f9e)

Is there any way to reconfigure this for the 433Mhz?

**Comment by pkendall64** [comment]
The frontend filtering will mean that it will not perform well. i.e. the filtering on the ES900 is deigned to filter out the frequencies outside the 868-915 range.

**Reply by krupis** [reply]
Understood.

I have gone through many available target devices in the ExpressLRS configurator:

![image](https://github.com/user-attachments/assets/086ca47e-dd1c-46c7-81da-3f9eb06f7a2b)


There does not seem to be a single device that I found that would allow setting anything other than:
![image](https://github.com/user-attachments/assets/79412b5e-6c46-43c3-a14d-ec3d660f48f7)


Is there any particular reason why 433Mhz is not so much supported and not widely popular?

You said that it would not perform well and that's fair. At this point I am just curious is there any way to change something to unlock other Regulatory domains apart from the 4 that I have just shown. For example, happymodel es900 rx has the following options:

![image](https://github.com/user-attachments/assets/f6e9747e-a47b-41d1-ac0b-6f2251041642)

Where are these options defined? Can I enable 433 just for testing purposes?

**Reply by gagarinlg** [reply]
It is very narrow, only 700kHz and is, at the same time, used for HAM radio applications, so it is prone to interference.

**Reply by krupis** [reply]
@gagarinlg 
Understood.

But how does setting Regulatory domains affect the receiver?
![image](https://github.com/user-attachments/assets/83b461fc-2c49-40f3-a9dd-cb49f8cd45e4)


if I wanted to test it out 433 Mhz on the ES900RX is the only option to download sources, setup build environment and manually change the user defines and set the Regulatory Domain to 433 as shown here:
![image](https://github.com/user-attachments/assets/5d783106-1495-4bba-b0fc-9f4b98e4f715)



Is that the only option? Why would there not be a choice left in the ExpressLRS to simplify this process to set the 433 Mhz band via the configurator directly?

**Reply by anti-vaxxer** [reply]
> Why would there not be a choice left in the ExpressLRS to simplify this process to set the 433 Mhz band via the configurator directly?

probably because as pkendall64 said the hardware is not designed for that frequency.

**Reply by gagarinlg** [reply]
Correct. There are RF frontend filters to filter harmonics in the TX direction and other transmissions in the RX direction. You will not get a good receiver sensitivity without hardware designed for that band and you might also create illegal unwanted emissions and multiples of the TX frequency.

**Comment by JyeSmith** [comment]
No existing SX1276 hardware will work as the Low Frequency input is a completely different pin.  Its not even connected to the RF front end and antenna!

![image](https://github.com/user-attachments/assets/c854a273-940d-462a-9f6c-abb7d80d7adb)

**Comment by SparkCD** [comment]
i have a E32-400M30S     its 400mhz rf and 30dbm    how to ues it for ELRS
